scripts/setup

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${MOS_DEBUG}" ]]; then
+    set -x
+fi
+
+function usage() {
+    echo -n \
+        "Usage: $(basename "$0")
+Attempts to setup the project's development environment.
+"
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
+    if [ "${1:-}" = "--help" ]; then
+        usage
+    else
+        docker-compose build
+        docker-compose run app npm install
+        docker-compose run app bower --allow-root install
```

Comment:
Oops, sorry, I should have called this out -- yes, this part should go in a `./scripts/update` and yes, I put it here to save time. I'll add a comment to that effect and open an issue so we don't forget in case this project sees more active development.